conf/modules/optical_flow_mateksys_3901_l0x.xml

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="mateksys_3901_l0x">
+  <doc>
+    <description>
+      Transfers flow X and Y and range data of the Mateksys3901-l0X sensor to the autopilot
+      This by the default serial connection not by connecting the debug pins on ther sensor
+      The Data is in MSP compatible output. If a MSP stack is availabe in the AP one could use that
+      In ths ase we just handle the byres comming from the sensor in MSP compatible handler for this sensor
+    </description>
+    <configure name="MATEKSYS_3901_L0X_PORT" value="UART1" description="Set to which UART the sensor serial stream pin is connected to"/>
+    <configure name="MATEKSYS_3901_L0X_BAUD" value="115200" description="Sets the baudrate of the UART"/>
+    
+  </doc>
+
+
+  <header>
+    <file name="mateksys_3901_l0x.h" dir="modules/optical_flow"/>
+  </header>
+  <init fun="mateksys3901l0x_init()"/>
+  <event fun="mateksys3901l0x_event()"/>
+  <!-- <periodic fun="mateksys3901l0x_downlink()"/> --> <!-- to get messages TODO-->
+  <makefile>
+
+    <!-- Configure default UART port and baudrate -->
+    <configure name="MATEKSYS_3901_L0X_PORT" default="UART3" case="upper|lower"/>
+    <configure name="MATEKSYS_3901_L0X_BAUD" default="B115200"/>
+
+    <!-- Enable UART and set baudrate -->
+    <define name="USE_$(MATEKSYS_3901_L0X_PORT_UPPER)"/>
+        <!--<define name="USE_$(MATEKSYS_3901_L0X_PORT)_TX" value="FALSE"/>-->
```

Comment:
Could you add a comment on when this define should be set to false, or mention this in the description of this define?

Reply:
@OpenUAS should I remove this one? It is still unchanged from the template we did at the beginning.

Reply:
You can add:
it the driver is used on a serial port where the TX pin is not available, since the TX is not needed for this sensor it can be disabled.